frontend/src/styles/_modules.scss

```diff
@@ -1,6 +1,7 @@
 @import 'modules/avatar';
 @import 'modules/buttons';
 @import 'modules/footer';
-@import 'modules/modals';
+@import 'modules/legal-information';
```

Comment:
Looks like these slipped back in - maybe needs a rebase? I think [jgruen's big header restyle PR pulled them out](https://github.com/mozilla/testpilot/commit/b29699b3b0cf9566935f9f15dc3bd860eabef0a3#diff-3cadb5852907eb1bbf57b5b2bd49db74L4)

Reply:
Looks like the `modules/legal-information` file wasn't deleted and isn't being used anywhere also